src/app/business/block/cloud-file-share/create/cloud-file-share-create.component.ts

```diff
@@ -99,6 +101,13 @@ export class CloudFileShareCreateComponent implements OnInit{
             'description': new FormControl('', {validators:[Validators.maxLength(250),Validators.pattern(this.validRule.description)]}),
             'type' : new FormControl('cloudFS')
         });
+
+        this.protocolTypeOptions = [
```

Comment:
Please add {
                label: 'CIFS',
                value: 'CIFS'
            }

Reply:
I have this ready. Since I did not receive any update I just retained NFS.